app/views/home/how_government_works.html.erb

```diff
@@ -1,534 +1,489 @@
 <% page_title "How government works" %>
 <% page_class "inside_gov_how-gov-works" %>
 
-<div id="how-government-works">
-  <header class="govuk-width-container">
-    <div class="govuk-grid-row">
-      <div class="govuk-grid-column-three-quarters">
+<div class="govuk-width-container">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <header>
```

Comment:
There is probably debate to be had about the pros / cons of multiple  `<header>`s on the page - however, following convention of other pages + as there's the `id="global-header"` on the page - I don't _think_ this is needed.